docs/sections/user_guide/cli/drivers/shared/help.rst

```diff
@@ -0,0 +1,13 @@
+.. literalinclude:: help.cmd
+   :language: text
+   :emphasize-lines: 1
+.. literalinclude:: help.out
+   :language: text
+
+All tasks take the same arguments. For example:
+
+.. literalinclude:: run-help.cmd
+   :language: text
+   :emphasize-lines: 1
+.. literalinclude:: run-help.out
+   :language: text
```

Comment:
Here's the factored-out help info that was (nearly) identical in each driver's `.rst` file. The reorganization of the driver-doc files made the `index.rst` file that includes this file and the driver's `.cmd` and `.out` files siblings, so what used to be e.g. `cdeps/help.cmd` can now just be `help.cmd`, such that the directory part of the filename is unneeded. Now this file can be used by each driver.